bin/receiver.py

```diff
@@ -179,25 +239,31 @@ def main():
         i = 0
         # The message listening loop.
         while True:
+            try:
+                time.sleep(1)
+                if protocol == 'HTTPS':
+                    # We need to pull down messages as part of
+                    # this loop when using HTTPS.
+                    ssm.pull_msg_rest()
+
+                if i % REFRESH_DNS == 0:
```

Comment:
Poss doesn't need to be in try...except. Check if affects ELK.

Reply:
Will leave it as there is the `send_ping` bit which could potentially raise an exception.